app/views/newflow/login_signup/educator_signup_form.html.erb

```diff
@@ -133,15 +145,20 @@
 
         <div class="content">
           <%= f.submit I18n.t(:"login_signup_form.continue_button"),
-                       data: { disable_with: I18n.t(:"login_signup_form.continue_button") },
+                       data: { disable_with: false },
```

Comment:
I just wanna double check that you meant to keep this change. Or, didn't you make this change as a test to try and fix the problem with the non-edu warning?